docs/current/sailbot_workspace/workflow.md

```diff
@@ -94,25 +123,32 @@ launch configueration.
 
 If you are having some trouble running our software, here are some things you can try:
 
-ROS:
-
-- Run the `clean` VS Code task to clean your build
-- Run the `purge` VS Code task to delete all generated files in the workspace
-
-Dev Container:
-
-- Run the `Dev Containers: Rebuild Container` command in the
-  [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
-  to rebuild the Dev Container
-
-VS Code:
-
-- Close and reopen the VS Code
-- Update VS Code and its extensions
-
-Docker:
-
-- Run `docker system prune` to remove all unused containers, networks, images (both dangling and unreferenced)
-    - Add `--all` to additionally remove unused images
-    - Add `--volumes` to additionally remove volumes
-    - Run `docker rmi -f $(docker images -aq)` to remove all images
+- Delete generated files
+    - Run the `clean` task to clean your build
+    - Run the `purge` task to delete all generated files
+- Rebuild the Dev Container: run the `Dev Containers: Rebuild Container` command
+- Reload VS Code: run the `Developer: Reload Window` command
+- Delete Docker files
+    - Run `docker system prune` to remove all unused containers, networks, images (both dangling and unreferenced)
+        - Add `--all` to additionally remove unused images (don't have a container associated with them)
+        - Add `--volumes` to additionally remove volumes (makes Bash history and ROS logs persist across containers)
+        - Run `docker rmi -f $(docker images -aq)` to remove all images
+
+    ??? tip "Running Docker CLI commands on Windows"
+
+        On Windows, Docker CLI commands should be run in the Ubuntu terminal while Docker Desktop is running.
+
+## Performance Issues
+
+If you are not satisfied with the performance of Sailbot Workspace, here are some things you can try:
+
+- Free up memory: close programs that you aren't using
+- Free up disk space: permanently delete large programs and files that you don't need anymore
+- Run Sailbot Workspace in a GitHub Codespace
+    - In a codespace with 8GB of RAM, building all packages from scratch with the `-q` argument takes about a minute.
+    If your computer takes longer than, or you want to free up memory and disk space, you can
+    [setup Sailbot Workspace in a GitHub Codespace](./setup.md#setup-sailbot-workspace-in-a-github-codespace){target=_blank}
+- If you are running Sailbot Workspace on Windows, dual boot Ubuntu and run Sailbot Workspace there
```

Comment:
Should we add some resources on dual boot?

Reply:
Good idea